.ahoy.yml

```diff
@@ -161,7 +161,7 @@ commands:
           || [ "${ALLOW_BDD_FAIL:-0}" -eq 1 ]
       else
         # run tests with the specified tag
-        ahoy cli "behave -k ${*:-test/features} --tags=$BEHAVE_TAG --tags=-OpenData --tags=-Publications" \
+        ahoy cli "behave -k ${*:-test/features} --tags=$BEHAVE_TAG" \
```

Comment:
so we moved to all tests based bon the $behave_tag?

Reply:
My bad, will revert this